src/Serilog.Sinks.Udp/Serilog.Sinks.Udp.csproj

```diff
@@ -1,5 +1,4 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
```

Comment:
Let me guess, you're using JetBrains Rider?

I'm experiencing the same issue when developing on Linux, and haven't figured out how to stop Rider from reformatting the project file. In any case, it doesn't matter, its just funny to me.